core/src/replay_stage.rs

```diff
@@ -293,6 +296,8 @@ impl ReplayStage {
                 let mut partition_exists = false;
                 let mut skipped_slots_info = SkippedSlotsInfo::default();
                 let mut replay_timing = ReplayTiming::default();
+                let mut voted_signatures = Vec::new();
```

Comment:
Why 201?

Reply:
I chose a 200 signature limit for how large this can be. I'm not sure we need to size it initially, most cases should not use the full 200 and the path isn't that performance sensitive.